src/pages/DeviceList.js

```diff
@@ -8,51 +8,51 @@ import VolumeSlider from '../components/VolumeSlider'
 async function setConfiguration (params) {
   const body = JSON.stringify(params)
   const headers = { 'Content-Type': 'application/json' }
-  await fetch('/api/configuration', { method: 'PATCH', body, headers })
+  await fetch('/api', { method: 'PUT', body, headers })
 }
 
-async function getDevices () {
+async function getConfiguration () {
   console.log('fetching!')
-  const res = await fetch('http://localhost:3000/api/devices')
+  const res = await fetch('/api')
```

Comment:
This new GET fat endpoint returns something like this:

```
{
  device: "Foobar",
  volume: 30,
  devices: [
    // ...
  ]
}
```

Reply:
Potentially we could also add so that you can make individual GET requests to `/devices`, `/device` and `/volume`, but for now this works better since we always want all the information.

Of course, GraphQL would solve all of these problems much more elegant, but it seems a bit overkill 😄